resources/views/templates/admin-user/index.blade.php

```diff
@@ -0,0 +1,112 @@
+{{'@'}}extends('brackets/admin-ui::admin.layout.default')
+
+{{'@'}}section('title', trans('admin.{{ $modelLangFormat }}.actions.index'))
+
+{{'@'}}section('body')
+
+    <{{ $modelJSName }}-listing
+        :data="{{'{{'}} $data->toJson() }}"
+        :activation="!!'@{{ $activation }}'"
+        :url="'{{'{{'}} url('admin/{{ $resource }}') }}'"
+        inline-template>
+
+        <div class="row">
+            <div class="col">
+                <div class="card">
+                    <div class="card-header">
+                        <i class="fa fa-align-justify"></i> {{'{{'}} trans('admin.{{ $modelLangFormat }}.actions.index') }}
+@if($export)
+                        <a class="btn btn-primary btn-sm pull-right m-b-0 ml-2" href="{{'{{'}} url('admin/{{ $resource }}/export') }}" role="button"><i class="fa fa-file-excel-o"></i>&nbsp; {{'{{'}} trans('admin.{{ $modelLangFormat }}.actions.export') }}</a>
+@endif
+                        <a class="btn btn-primary btn-spinner btn-sm pull-right m-b-0" href="{{'{{'}} url('admin/{{ $resource }}/create') }}" role="button"><i class="fa fa-plus"></i>&nbsp; {{'{{'}} trans('admin.{{ $modelLangFormat }}.actions.create') }}</a>
+                    </div>
+                    <div class="card-block" v-cloak>
```

Comment:
tu treba classu card-body namiesto card-block
```suggestion
                    <div class="card-body">
```